met-api/src/met_api/models/staff_user.py

```diff
@@ -60,6 +60,11 @@ def get_user_by_external_id(cls, _external_id) -> StaffUser:
         """Get a user with the provided external id."""
         return cls.query.filter(func.lower(StaffUser.external_id) == func.lower(_external_id)).first()
 
+    @classmethod
+    def get_user_by_id(cls, _user_id) -> StaffUser:
+        """Get a user with the provided external id."""
+        return cls.query.filter(StaffUser.id == _user_id).first()
+
```

Comment:
if it inherits from basemodel , get_by_id method shud also be inherited..this wont be needed

Reply:
Thanks, removed